easybuild/easyconfigs/p/PBZIP2/1.1.8-ictce-6.2.5.eb

```diff
@@ -0,0 +1,34 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Pablo Escobar Lopez
+# Swiss Institute of Bioinformatics
+# Biozentrum - University of Basel
+
+easyblock = 'MakeCp'
+
+name = 'PBZIP2'
+version = '1.1.8' 
+
+homepage = 'http://compression.ca/pbzip2/'
+description = """ PBZIP2 is a parallel implementation of the bzip2 block-sorting file 
+ compressor that uses pthreads and achieves near-linear speedup on SMP machines. The output 
+ of this version is fully compatible with bzip2 v1.0.2 or newer (ie: anything compressed 
+ with pbzip2 can be decompressed with bzip2)."""
+
+toolchain = {'name': 'ictce', 'version': '6.2.5'} 
+
+source_urls = ['http://compression.ca/pbzip2/']
+sources = [SOURCELOWER_TAR_GZ]
+
+makeopts = ' CC=$CXX'
+
+parallel = 1
+
+files_to_copy = [(["pbzip2"], "bin"), (["pbzip2.1"], "man/man1"), "README", "ChangeLog"]
+        
+sanity_check_paths = {
+    'files': ["bin/pbzip2"],
+    'dirs': [""],
```

Comment:
just drop "" and the extra line at the end; these look good and very useful otherwise!